html/templates/header.html

```diff
@@ -34,15 +34,9 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/tgstation?tab=members'>-Click Here-</a><br></font>
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/tgstation/-tg-station/graphs/contributors'>-Click Here-</a><br></font>
```

Comment:
Hey hey, just replace the link with https://github.com/Citadel-Station-13/Citadel-Station-13/graphs/contributors. Don't remove this line.